lib/arithmetic.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const arithmetic = module.exports = {};
+
+arithmetic.add = function add (a, b) {
+  return a + b;
```

Comment:
Remember that the instructions said your arithmetic functions should check if the arguments are numbers.